### PR TITLE
KHWE-55: Create About Page

### PR DESCRIPTION
**Pull request by pavelbaranchuk**
about

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link
```
Comment (ihor4568) [inline public/index.html]: these fonts are already here

File: src/components/App.js [hunk 2]
```diff
@@ -23,14 +20,29 @@ class App extends Component {
     classes: PropTypes.object.isRequired
   };
 
+  state = {
+    itemMount: "Albums"
```
Comment (ihor4568) [inline src/components/App.js]: can you rename this to smth like `currentTab` or `currentPage` ??

File: src/components/about/About.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  link: {
+    textDecoration: "none",
+    fontSize: "1.35rem"
+  },
+  mentor: {
+    padding: "2rem 0 1rem"
+  }
+});
+
+class About extends Component {
```
Comment (ihor4568) [inline src/components/about/About.js]: you can use functional component here

File: src/components/about/AboutCard.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  title: {
+    textAlign: "center",
+    padding: "1rem 0 1rem 0"
+  },
+  link: {
+    textDecoration: "none"
+  }
+};
+
+function AboutCard(props) {
```
Comment (ihor4568) [inline src/components/about/AboutCard.js]: you can use functional component here

File: src/components/albums/Albums.js [hunk 5]
```diff
@@ -73,7 +76,7 @@ class Albums extends Component {
     const { classes } = this.props;
     return (
       <>
-        <Typography variant="h4" component="h2">
+        <Typography variant="h4" component="h2" className={classes.title}>
```
Comment (ihor4568) [inline src/components/albums/Albums.js]: can you extract title as a separate component? and reuse it in all components?

File: src/components/shared/Drawer.js [hunk 6]
```diff
@@ -124,7 +126,7 @@ class DrawerComponent extends Component {
         </List>
         <Divider />
         <List>
-          <ListItem button>
+          <ListItem button onClick={this.props.onAboutClick}>
```
Comment (ihor4568) [inline src/components/shared/Drawer.js]: no - there should be one method for each click handler - like onItemClick. And then in each on click you call it with different parameter like this:


```
<ListItem button onClick={() => this.props.onItemClick('about')}>

<ListItem button onClick={() => this.props.onItemClick('albums')}>

...

```

File: src/components/shared/Main.js [hunk 7]
```diff
@@ -22,16 +27,19 @@ const styles = theme => ({
...
   render() {
     const { classes } = this.props;
+
     return (
       <div className={classes.main}>
         <main className={classes.content}>
           <div className={classes.toolbar} />
-          <Albums />
+          {this.props.itemMount === "About" ? <About /> : null}
+          {this.props.itemMount === "Albums" ? <Albums /> : null}
```
Comment (ihor4568) [inline src/components/shared/Main.js]:
```
const { currentTab } = this.props


return (
...

{currentTab === "about" && <About />}
{currentTab === "albums" && < Albums />}

...
)
```

File: src/components/shared/Main.js [hunk 8]
```diff
@@ -1,11 +1,15 @@
...
 import PropTypes from "prop-types";
 
 const styles = theme => ({
   main: {
-    paddingBottom: "6.5rem"
+    paddingBottom: "6.5rem",
+    height: "100%",
+    maxWidth: "80rem",
+    overflow: "hidden",
+    boxSizing: "border - box",
```
Comment (pavelbaranchuk) [inline src/components/shared/Main.js]: ?

File: src/components/shared/Title.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+import { withStyles } from "@material-ui/core/styles";
+
+const styles = () => ({
+  title: {
+    padding: "1rem 0 1.5rem"
+  }
+});
+
+function Title(props) {
+  const { classes } = props;
```
Comment (pavelbaranchuk) [inline src/components/shared/Title.js]: ({ . })

File: src/components/App.js [hunk 10]
```diff
@@ -29,6 +29,14 @@ class App extends Component {
     classes: PropTypes.object.isRequired
   };
 
+  state = {
```
Comment (pavelbaranchuk) [inline src/components/App.js]: remove

File: public/index.html [hunk 11]
```diff
@@ -3,16 +3,16 @@
...
+      rel="stylesheet"
```
Comment (pavelbaranchuk) [inline public/index.html]: remove

File: package.json [hunk 12]
```diff
@@ -10,9 +10,10 @@
     "prop-types": "15.6.2",
     "react": "16.6.3",
     "react-dom": "16.6.3",
-    "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.1",
-    "styled-components": "4.1.1"
+    "styled-components": "4.1.1",
```
Comment (pavelbaranchuk) [inline package.json]: remove

**Pull request merged**